gen/generate.go

```diff
@@ -165,6 +169,43 @@ func Generate(m *compile.Module, o *Options) error {
 	return nil
 }
 
+func isDuplicateFileAfterNormalization(f string) error {
+	absFileName, err := filepath.Abs(f)
+	if err != nil {
+		return generateError{Name: f, Reason: fmt.Errorf("File %q does not exist: %v", f,
+			err)}
+	}
+	normalizedFileName := filePathWithUnderscore(absFileName)
+	if fileNormalized(absFileName, normalizedFileName) && fileExists(normalizedFileName) {
+		return generateError{Name: f,
+			Reason: fmt.Errorf("File after normalization %q is colliding with existing file %q in the path, with error: %v",
```

Comment:
- go errors always start lower-case (I think `go vet` should catch this 🤔 ?)
- we can simplify the error message, and should re-order the arguments
- the `err` value used here is not relevant and will always be `nil`
```suggestion
			Reason: fmt.Errorf("normalizing %q collides with existing file %q", absFileName, normalizedFileName)
```